docs/reference/sql/limitations.asciidoc

```diff
@@ -184,7 +184,7 @@ setting needs to be adjusted or fewer columns retrievable from `docvalue_fields`
 [[aggs-in-pivot]]
 === Aggregations in the <<sql-syntax-pivot, `PIVOT`>> clause
 
-The aggregation expression in <<sql-syntax-pivot, `PIVOT`>> will currently accept only one aggregation. It is thus not possible to obtain multiple aggregations for any one pivoted columns.
+The aggregation expression in <<sql-syntax-pivot, `PIVOT`>> will currently accept only one aggregation. It is thus not possible to obtain multiple aggregations for any one pivoted column.
```

Comment:
Have the feeling that `one` is superfluous but might be wrong.

Reply:
I believe that the `for any one <item>` should be a valid phrasing.